hw/drivers/trng/src/mbed_entropy_alt.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <trng/trng.h>
+#include "mbedtls/config_mynewt.h"
+
+#ifdef MBEDTLS_ENTROPY_HARDWARE_ALT
```

Comment:
I would avoid including `mbedtls/config_mynewt.h` and using instead `#if MYNEWT_VAL(MBEDTLS_ENTROPY_HARDWARE_ALT)` here.

Reply:
ah thats a good point